### ISSUE TITLE
Base image coverage for PHP 7.2.31, 7.3.18, and 7.4.6

### ISSUE THREAD

**Issue by bryangruneberg** [uselagoon/lagoon#1885]
**Describe the bug**
PHP 7.2.31, 7.3.18, and 7.4.6 have been released. The upstream images have not yet been updated (last updated 5/7/2020 at 5:16 am)

**To Reproduce**
Check the URLs on Docker Hub
1. https://hub.docker.com/_/php?tab=tags&page=1&name=7.2-fpm-alpine3.11
2. https://hub.docker.com/_/php?tab=tags&page=1&name=7.3-fpm-alpine3.11
3. https://hub.docker.com/_/php?tab=tags&page=1&name=7.4-fpm-alpine3.11

**Expected behavior**
The respective PHP versions are up to date with 7.2.31, 7.3.18, and 7.4.6

**Additional context**
This issue can be closed once the Docker Hub issues are up to date.

**Comment by bryangruneberg** [comment]
Quick hack script to check the versions against expectations: https://gist.github.com/bryangruneberg/d7dcfd549cb2e3a0f448d6981121f695

**Comment by bryangruneberg** [comment]
The alpine images have been updated

**Comment by bryangruneberg** [comment]
The amazeeio/php:7.{2,3,4} images are still using the old alpine base, so I think those need to be rebuilt.

**Comment by tobybellwood** [comment]
Yup, sorry! unfortunately the amazeeio images require a Lagoon release to update the PHP version (the PHP layers are "fixed" when the image is created).  We've got one coming up later this month, so it won't be too long.

This is an area where were trying to reduce some of the overhead, watch this space!

**Comment by tobybellwood** [comment]
done - released in Lagoon [1.6.0](https://github.com/amazeeio/lagoon/releases/tag/v1.6.0)
```
tobypop-os:~/sites/drupal$ docker run amazeeio/php:7.4-fpm printenv | grep VERSION
LAGOON_VERSION=v1.6.0
PHP_VERSION=7.4.6
NEWRELIC_VERSION=9.10.1.263
toby@pop-os:~/sites/drupal$ docker run amazeeio/php:7.3-fpm printenv | grep VERSION
LAGOON_VERSION=v1.6.0
PHP_VERSION=7.3.18
NEWRELIC_VERSION=9.10.1.263
toby@pop-os:~/sites/drupal$ docker run amazeeio/php:7.2-fpm printenv | grep VERSION
LAGOON_VERSION=v1.6.0
PHP_VERSION=7.2.31
NEWRELIC_VERSION=9.10.1.263
```
